.github/workflows/ci.yml

```diff
@@ -22,22 +22,25 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 2
-
+      - name: Set up pnpm 8.x
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
       - name: Set up Node.js environment
         uses: actions/setup-node@v3
         with:
           node-version: 18
-          cache: yarn
+          cache: pnpm
 
       - name: Install dependencies
-        run: yarn install
+        run: pnpm install --ignore-scripts
```

Comment:
When I don't include this `pnpm install` runs the `prepare` command in the `makeswift` package. This just double builds
<img width="317" alt="CleanShot 2023-10-10 at 15 47 56@2x" src="https://github.com/makeswift/makeswift/assets/20950876/6aaabd26-ff66-4a05-a889-3cefa3353d7b">


Not sure why yarn didn't/doesn't do this.